testdata/templates/service/repos.tpl

```diff
@@ -0,0 +1,604 @@
+{{- define "service_repo_node_1" }}
```

Comment:
It would be really nice to DRY up this data and remove the weird references to things not part of our organization - but this is a minor tweak and i'm fine shipping with out it.  Just a comment when creating future test data.  Keep it DRY, clean, and reuseable.